test/unit/config.spec.ts

```diff
@@ -2,7 +2,7 @@ import { defaultConfig } from "src/config"
 
 describe("config.ts", () => {
   describe("defaultConfig", () => {
-    it("should have hedging and name properties", () => {
+    it("has hedging and name properties", () => {
```

Comment:
This shouldn't be a test but a type.

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/340/commits/ccdb806d4ae012f1c95fb0803b6120b1c3cc8509